src/libraries/Microsoft.Extensions.Logging.EventSource/src/LoggingEventSource.cs

```diff
@@ -505,6 +554,8 @@ private static unsafe void SetEventData<T>(ref EventData eventData, ref T value,
             {
                 string str = (value as string)!;
 #if DEBUG
+                Debug.Assert(str != null);
```

Comment:
why did you remove this assert?

Reply:
I added it 😄

Reply:
Oh, somehow the diff in the tool came red. or I was dreaming :-)